nvflare/app_common/executors/script_executor.py

```diff
@@ -34,62 +29,51 @@
-            task_script_args=task_script_args,
-            task_wait_time=task_wait_time,
-            result_pull_interval=result_pull_interval,
-            train_with_evaluation=train_with_evaluation,
-            train_task_name=train_task_name,
-            evaluate_task_name=evaluate_task_name,
-            submit_model_task_name=submit_model_task_name,
-            from_nvflare_converter_id=from_nvflare_converter_id,
-            to_nvflare_converter_id=to_nvflare_converter_id,
-            params_exchange_format=params_exchange_format,
-            params_transfer_type=params_transfer_type,
-            log_pull_interval=log_pull_interval,
-        )
+        self._script = script
+        self._script_args = script_args
+        self._launch_external_process = launch_external_process
+        self._params_transfer_type = params_transfer_type
+        self._params_exchange_format = params_exchange_format
+
+        self._from_nvflare_converter = None
+        self._to_nvflare_converter = None
+
         if torch_ok:
             if params_exchange_format == ExchangeFormat.PYTORCH:
-                fobs.register(TensorDecomposer)
-
-                if self._from_nvflare_converter is None:
-                    self._from_nvflare_converter = NumpyToPTParamsConverter(
-                        [AppConstants.TASK_TRAIN, AppConstants.TASK_VALIDATION]
-                    )
-                if self._to_nvflare_converter is None:
-                    self._to_nvflare_converter = PTToNumpyParamsConverter(
-                        [AppConstants.TASK_TRAIN, AppConstants.TASK_SUBMIT_MODEL]
-                    )
+                self._from_nvflare_converter = NumpyToPTParamsConverter(
+                    [AppConstants.TASK_TRAIN, AppConstants.TASK_VALIDATION]
+                )
+                self._to_nvflare_converter = PTToNumpyParamsConverter(
+                    [AppConstants.TASK_TRAIN, AppConstants.TASK_SUBMIT_MODEL]
+                )
         if tf_ok:
             if params_exchange_format == ExchangeFormat.NUMPY:
-                if self._from_nvflare_converter is None:
-                    self._from_nvflare_converter = NumpyToKerasModelParamsConverter(
-                        [AppConstants.TASK_TRAIN, AppConstants.TASK_VALIDATION]
-                    )
-                if self._to_nvflare_converter is None:
-                    self._to_nvflare_converter = KerasModelToNumpyParamsConverter(
-                        [AppConstants.TASK_TRAIN, AppConstants.TASK_SUBMIT_MODEL]
-                    )
+                self._from_nvflare_converter = NumpyToKerasModelParamsConverter(
+                    [AppConstants.TASK_TRAIN, AppConstants.TASK_VALIDATION]
+                )
+                self._to_nvflare_converter = KerasModelToNumpyParamsConverter(
+                    [AppConstants.TASK_TRAIN, AppConstants.TASK_SUBMIT_MODEL]
+                )
         # TODO: support other params_exchange_format
```

Comment:
if no self._from_nvflare_converter and no self._to_nvflare_converter, what will happen?